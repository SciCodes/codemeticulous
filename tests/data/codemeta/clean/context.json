{
  "@context": {
    "schema": "http://schema.org/",
    "codemeta": "https://w3id.org/codemeta/3.0"
  },
  "schema:name": "Example Software",
  "codemeta:author": {
    "@type": "Person",
    "name": "Jane Doe"
  },
  "type": "SoftwareSourceCode"
}
