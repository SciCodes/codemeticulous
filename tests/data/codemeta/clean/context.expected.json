{
  "@context": "https://w3id.org/codemeta/3.0",
  "@type": "SoftwareSourceCode",
  "name": "Example Software",
  "author": {
    "@type": "Person",
    "name": "Jane Doe"
  }
}
